layouts/workshop/single.html

```diff
@@ -0,0 +1,112 @@
+{{ define "main" }}
+
+{{ $path := split $.Source.File.Path .Site.Params.PathSeparator }}
+{{ $event_slug := index $path 1 }}
+{{ $e :=  (index $.Site.Data.events $event_slug) }}
+{{ $city_slug := substr $e.name 5 }}
+
+
+<div class = "row">
+    <div class = "col-md-5 offset-md-1">
+    <h2 class="talk-page">{{ .Title }}</h2>
+
+    {{ if isset .Params "youtube" }}
```

Comment:
Is it likely that a workshop would ever have a youtube video of it? Can we remove this logic, and then likewise remove it from the frontmatter?